teku/src/main/java/tech/pegasys/teku/cli/BeaconNodeCommand.java

```diff
@@ -326,6 +326,8 @@ public void setLogLevels() {
       // set log level per CLI flags
       LoggingConfigurator.setAllLevels(logLevel);
     }
+    // jupnp logs a lot of context to level WARN, and it is quite verbose.
+    LoggingConfigurator.setAllLevels("org.jupnp", Level.ERROR);
```

Comment:
I'm pretty nervous about this.  The only noisy logs I saw were from `SpecificationViolationReporter` which can be disabled via a static method which would be preferable.

Reply:
it's not just that though, i get spammed with 
```
 RetrieveRemoteDescriptors | Device descriptor retrieval failed: http://192.168.1.2:8080/upnp, 404 
```
every minute, it's the one i thought was really noisy...